core/src/test/java/org/apache/iceberg/rest/responses/TestLoadTableResponseParser.java

```diff
@@ -138,8 +137,8 @@ public void roundTripSerdeV1() {
   }
 
   @ParameterizedTest
-  @ValueSource(ints = {2, MAX_FORMAT_VERSION})
-  public void roundTripSerdeV2andHigher(int formatVersion) {
+  @ValueSource(ints = 3)
```

Comment:
I would change this to stream  V3 -> MAX_FORMAT_VERSION , inclusive . Just trying to forward protect this

Reply:
I've used the approach above and standardized on `@FieldSource("org.apache.iceberg.TestHelpers#ALL_VERSIONS")` with an assumption that `formatVersion >= 3`.